.vscode/settings.json

```diff
@@ -127,12 +124,6 @@
                 "${workspaceFolder}/build": true
         },
         "search.showLineNumbers": true,
-        "telemetry.enableTelemetry": false,
-        "terminal.integrated.scrollback": 5000,
```

Comment:
The default scrollback is pretty small (shorter than the output of some builds, if anything I would increase this even more. What's your particular issue?

Reply:
During development, I may want to use a much larger scrollback buffer than what is proposed in this configuration. So I would also leave this choice to the user.

Reply:
I'm not aware of anyone that doesn't want that. Let's just bump it to save some time instead of having a crappy experience by default

Reply:
How much, do you think, is sufficient? 15000?

Reply:
No idea, but 15000 sounds fine to me if that's enough for you. This is all trivial amounts of memory for a development machine.

Reply:
Done. Set to 15000.